docs/containers.conf.5.md

```diff
@@ -0,0 +1,339 @@
+
+# FORMAT
+The [TOML format][toml] is used as the encoding of the configuration file.
+Every option is nested under its table. No bare options are used. The format of
+TOML can be simplified to:
+
+    [table1]
+    option = value
+
+    [table2]
+    option = value
+
+    [table3]
+    option = value
+
+    [table3.subtable1]
+    option = value
+
+## CONTAINERS TABLE
+The containers table contains settings pertaining to the OCI runtime that can
+configure and manage the OCI runtime.
+
+**additional_devices**=[]
+  List of additional devices.
+  Specified as 'device-on-host:device-on-container:permissions',
+  for example: "--additional-devices=/dev/sdc:/dev/xvdc:rwm".
+
+**additional_volumes**=[]
+  List of additional volumes.
+  Specified as "directory-on-host:directory-in-container:options",
+  for example:  "--aditional-volumes=/db:/var/lib/db:ro".
+
+**apparmor_profile**=""
+  Used to change the name of the default AppArmor profile of container engines.
+The default profile name is "container-default".
+
+**cgroupns**="private"
+  Default way to to create a cgroup namespace for the container
+  Options are:
+    `private` Create private Cgroup Namespace for the container.
+    `host`    Share host Cgroup Namespace with the container.
+
+**cgroup_manager**="systemd"
+  The cgroup management implementation used for the runtime. Supports `cgroupfs`
+and `systemd`.
+
+**default_capabilities**=[]
+  List of default capabilities for containers.
+
+  The default list is:
+```
+  default_capabilities = [
+	"AUDIT_WRITE",
+	      "CHOWN",
+	      "DAC_OVERRIDE",
+	      "FOWNER",
+	      "FSETID",
+	      "KILL",
+	      "MKNOD",
+	      "NET_BIND_SERVICE",
+	      "NET_RAW",
+	      "SETGID",
+	      "SETPCAP",
+	      "SETUID",
+	      "SYS_CHROOT",
+  ]
+```
+
+**default_sysctls**=[]
+  List of default sysctls.
+
+**default_ulimits**=[]
+  A list of ulimits to be set in containers by default,
+  specified as "ulimit name=soft-limit:hard-limit",
+  for example:"nofile=1024:2048".
+
+**dns_options**=[]
+  List of default DNS options to be added to /etc/resolv.conf inside of the
+container.
+
+**dns_searches**=[]
+  List of default DNS search domains to be added to /etc/resolv.conf inside of
+the container.
+
+**dns_servers**=[]
+  A list of dns servers to override the DNS configuration passed to the
+container. The special value “none” can be specified to disable creation of
+/etc/resolv.conf in the container.
+
+**env**=["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
+  Environment variable list for the container process, used for passing
+environment variables to the container.
+
+**env_host**=false
+  Pass all host environment variables into the container.
+
+**hooks_dir**=["/etc/containers/oci/hooks.d", ...]
+  Path to the OCI hooks directories for automatically executed hooks.
+
+**http_proxy**=[]"`
+  The proxy environment variable list to apply to container process
```

Comment:
Missing trailing dot

Reply:
Maybe _"The proxy environment within containers."_. Might be a matter of taste but the _within_ makes it more obvious to me that the env var applies to the container and not the engine.

Reply:
Added a bunch more text explaining what http_proxy is